pkg/gui.py

```
@@ -190,152 +203,123 @@ def check_status(self, thread):
+            self.add_to_history(message_item)
 
-            msg = item[0]
-            display_info = item[1]
-            print("GUI 1: ", msg)
-
-            self.popup_widget.geometry('+%d+%d' % (0, 0))
+            msg = message_item[0]
+            logger.debug(f"Popping up message: {msg}")
 
+            self.popup_widget.geometry('+0+0')
             self.popup_widget.attributes('-topmost', True)
```

Ruff (0.8.0)

Boolean positional value in function call

(FBT003)